org-noter-core.el

```diff
@@ -1845,6 +1845,26 @@ See `org-noter-notes-window-behavior' for more information."
               (org-entry-put nil org-noter--property-behavior (format "%s" chosen-behaviors))
             (org-entry-delete nil org-noter--property-behavior))))))))
 
+(defun org-noter-toggle-notes-window-location ()
+  "Toggle between side- and bottom-notes window location.
+Only acts on the current session."
+  (interactive)
+  (org-noter--with-valid-session
+   (let ((current-notes-location (org-noter--session-window-location session))
+         (notes-buffer (org-noter--session-notes-buffer session)))
+     (cond ((eq current-notes-location 'horizontal-split)
+            (setf (org-noter--session-window-location session) 'vertical-split))
+           ((eq current-notes-location 'vertical-split)
+            (setf (org-noter--session-window-location session) 'horizontal-split)))
+     (let (exists)
+       (dolist (window (get-buffer-window-list notes-buffer nil t))
+         (setq exists t)
+         (with-selected-frame (window-frame window)
+           (if (= (count-windows) 1)
+               (delete-frame)
+             (delete-window window))))
+       (when exists (org-noter--get-notes-window 'force))))))
```

Comment:
@dmitrym0: you could argue that the chunk starting from L1859 should be its own function, since it is lifted straight from `org-noter-set-notes-window-location`.  But without prodding, I take the easy route...